plugins/none-ls/default.nix

```diff
@@ -28,6 +28,15 @@ in {
         description = "Plugin to use for none-ls";
       };
 
+      enableLspFormat = mkOption {
+        type = types.bool;
+        default = false;
```

Comment:
we could maybe set the default to `plugins.lsp-format.enable` indeed?

Reply:
Yes this makes sense.

Reply:
Done.